functions/go/starlark/README.md

```diff
@@ -1,18 +1,25 @@
 # starlark
 
-### Overview
+## Overview
 
 <!--mdtogo:Short-->
 
 The `starlark` function contains a Starlark interpreter to run a Starlark script
 to mutate or validate resources.
 
+The starlark script is so flexible to do many things such as:
+
+- Set namespace for all KRM resources.
```

Comment:
Let's avoid use-cases that we have well defined functions for. May be list one that we recently ran into:

`Add an annotation on the basis of a condition`